lib/msal-core/package.json

```diff
@@ -85,6 +87,7 @@
     "typedoc": "^0.14.2",
     "typescript": "^3.2.1",
     "uglifyjs-webpack-plugin": "^2.1.1",
+    "uuid": "^3.3.2",
```

Comment:
Seems like this should be a regular dependency?

Reply:
good call